rossum_api/api_client.py

```diff
@@ -99,7 +100,7 @@ def __init__(
         self.username = username
         self.password = password
         self.token = token
-        self.client = httpx.AsyncClient()
+        self.client = httpx.AsyncClient(timeout=timeout)
```

Comment:
Is it a good idea to turn off the timeout by default? Hanging the connection indefinitely does not seem like a great default. How about just increasing the timeout to smt like 60s?

Reply:
Let's not make any assumptions about a good timeout value in the library. Requests library also doesn't have a timeout.